cdap-ui/app/cdap/components/AbstractWidget/FormInputs/Select/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import Select from '@material-ui/core/Select';
+import withStyles, { WithStyles } from '@material-ui/core/styles/withStyles';
+import InputBase from '@material-ui/core/InputBase';
+
+const styles = (theme) => {
+  return {
+    input: {
+      borderRadius: 4,
+      position: 'relative' as 'relative',
+      border: '1px solid #ced4da',
+      padding: '10px 18px 10px 12px',
+      '&:focus': {
+        backgroundColor: 'transparent',
+        borderRadius: 4,
+      },
+    },
+  };
+};
+const CustomizedInput = withStyles(styles)(InputBase);
+
+interface ISelectOptions {
+  value: string | number; // We need to expand this when we have complex use cases
+  label: string;
+}
+interface ISelectProps {
+  value: string;
+  onChange: (value: string) => void;
+  options: ISelectOptions[];
+  init: string; // initial selection?
```

Comment:
can we call this `defaultValue`?

Reply:
Have removed this. default value will be passed as value when no value is set by the user (initial render). We don't need a separate value for it.